ospf6d/ospf6_top.c

```diff
@@ -644,11 +644,12 @@ DEFUN (no_ospf6_distance_source,
 
 DEFUN (ospf6_interface_area,
        ospf6_interface_area_cmd,
-       "interface IFNAME area A.B.C.D",
+       "interface IFNAME area <A.B.C.D|(0-4294967295)>",
```

Comment:
The area ID argument was unintentionally made optional in the first set of these diffs - hoping this will address the CI failure.

Reply:
Looks like that fixed it. `interface IFNAME` from ospf6 mode now switches to interface mode as before.